LayoutTests/fast/forms/number/number-stepup-stepdown-expected.txt

```diff
@@ -1,174 +1,64 @@
-Check stepUp() and stepDown() bahevior for number type.
+Check stepUp() and stepDown() behavior for number type.
 
 On success, you will see a series of "PASS" messages, followed by "TEST COMPLETE".
 
-
 Number type
-
 Invalid value
 PASS stepUp("", null, null) is "1"
 PASS stepDown("", null, null) is "-1"
-
 Non-number arguments
 PASS stepUp("0", null, null, "0") is "0"
 PASS stepDown("0", null, null, "0") is "0"
 PASS stepUp("0", null, null, "foo") is "0"
 PASS stepDown("0", null, null, "foo") is "0"
 PASS stepUp("0", null, null, null) is "0"
 PASS stepDown("0", null, null, null) is "0"
-
 Normal cases
 PASS stepUp("0", null, null) is "1"
 PASS stepUp("1", null, null, 2) is "3"
 PASS stepUp("3", null, null, -1) is "2"
 PASS stepDown("2", null, null) is "1"
 PASS stepDown("1", null, null, 2) is "-1"
 PASS stepDown("-1", null, null, -1) is "0"
-
-Fractional cases
```

Comment:
Why are we deleting these test cases? Can we rebaseline instead?

Reply:
Cool, let me do it. Thanks!

Reply:
Fixed.